### PR TITLE
better logging with login from multiple IP

### PR DESCRIPTION
**Pull request by sanderjo**
Closes https://github.com/sabnzbd/sabnzbd/discussions/2354

### CODE REVIEW

**Comment by Safihre** [comment]
Could you share a screenshot of the new 1 liner Warning? It seems to me we might need to change the wording maybe.

**Comment by sanderjo** [comment]
image

**Comment by puzzledsab** [comment]
I suggest removing "(too many)" and maybe use %s to place the address. Having it at the end may not be correct for all languages.

**Comment by sanderjo** [comment]
So how about "Multiple IP login"? Not 100% correct, but short

File: sabnzbd/downloader.py [hunk 1]
```diff
@@ -853,13 +853,19 @@ def __finish_connect_nw(self, nw: NewsWrapper) -> bool:
                     self.plan_server(server, _PENALTY_TOOMANY)
                     server.threads -= 1
             elif error.code in (502, 481, 482) and clues_too_many_ip(error.msg):
-                # Account sharing?
+                # Login from (too many) different IP addresses
                 if server.active:
-                    errormsg = T("Probable account sharing") + display_msg
+                    errormsg = T("Login from (too many) different IP addresses") + display_msg
+                    logging.debug("SJ100 %s", display_msg)
```
Comment (Safihre) [inline sabnzbd/downloader.py:859]: Remove 😉

File: sabnzbd/downloader.py [hunk 2]
```diff
@@ -853,13 +853,19 @@ def __finish_connect_nw(self, nw: NewsWrapper) -> bool:
                     self.plan_server(server, _PENALTY_TOOMANY)
                     server.threads -= 1
             elif error.code in (502, 481, 482) and clues_too_many_ip(error.msg):
-                # Account sharing?
+                # Login from (too many) different IP addresses
                 if server.active:
-                    errormsg = T("Probable account sharing") + display_msg
+                    errormsg = T("Login from (too many) different IP addresses") + display_msg
```
Comment (Safihre) [inline sabnzbd/downloader.py:858]: Agree partially with @puzzledsab, let's make it `Login from too many different IP addresses to server %s`.

**Comment by Safihre** [comment]
I made some small textual changes to the wiki text and 2 small comments, then we can merge it.
After that I will probably improve this code for all these errors, such ugly IF's!

**Comment by sanderjo** [comment]
Message is now

image

**Comment by sanderjo** [comment]
solves https://github.com/sabnzbd/sabnzbd/discussions/2354

**Pull request merged**